test/snippet/io/alignment_file/alignment_file_input_construction_without_automatic_type_deduction.cpp

```diff
@@ -0,0 +1,35 @@
+#include <sstream>
+#include <string>
+
+#include <seqan3/io/alignment_file/input.hpp>
+#include <seqan3/core/type_list.hpp>
+
+int main()
+{
+    std::string input
+    {
+        "@HD\tVN:1.6\tSO:coordinate\n"
+        "r001\t99\tref\t7\t30\t8M2I4M1D3M\t=\t37\t39\tTTAGATAAAGGATACTG\t*\n"
+    };
+    std::istringstream iss(input);
+
+    seqan3::alignment_file_input<seqan3::alignment_file_input_default_traits<>,  // The expected format.
+                                 seqan3::fields<seqan3::field::SEQ,              // The default types; you can adjust this list
```

Comment:
please outsource the filed types so the class definition does not become so bloated.